db/migrate/20200922151556_add_peer_to_peer_matching.rb

```diff
@@ -0,0 +1,5 @@
+class AddPeerToPeerMatching < ActiveRecord::Migration[6.0]
+  def change
+    add_column :system_settings, :allow_peer_to_peer_matching, :boolean, null: false, default: false
+  end
+end
```

Comment:
Oops, we should've given you this information up front, but there's an existing setting for this, stored in the [`exchange_type`](https://github.com/rubyforgood/mutual-aid/pull/730/files#diff-1acd2e7e27a227829d5d14a91c863bb6R429) column. The [existing options](https://github.com/rubyforgood/mutual-aid/blob/main/app/models/system_setting.rb#L3-L9) support a range of exchange types, but i think for now it should be sufficient to check `System_setting.peer_to_peer?`.

Apologies for not catching this earlier! 😬

Reply:
Not a problem. I have made necessary changes and using `SystemSetting.current_settings&.peer_to_peer?` now.